src/status_im/ui/screens/wallet/views.cljs

```diff
@@ -90,8 +90,8 @@
                           [react/refresh-control {:on-refresh #(re-frame/dispatch [:update-wallet])
                                                   :tint-color :white
                                                   :refreshing false}])}
-      [react/view {:style styles/scroll-top}] ;; Hack to allow different colors for top / bottom scroll view]
       [total-section portfolio-value currency]
       [list/action-list actions
        {:container-style styles/action-section}]
-      [asset-section assets currency]]]))
+      [asset-section assets currency]
+      [react/view {:style styles/scroll-bottom}]]])) ;; Hack to allow different colors for top / bottom scroll view
```

Comment:
I understand that it wasn't added in this PR, but @psdp can you please put this comment above `[react/view {:style styles/scroll-bottom}]`?

Reply:
:ok_hand: done.